content/release-notes/2022-10-25.md

```diff
@@ -0,0 +1,16 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Add support of PostgreSQL 15.0 and its PostgreSQL extensions.
```

Comment:
```suggestion
- Compute: Added support for PostgreSQL 15.0 and PostgreSQL 15 extensions supported by Neon. You can now select either PostgreSQL 14 or PostgreSQL 15 when creating a Neon project. 
```

Reply:
>You can now select either PostgreSQL 14 or PostgreSQL 15 when creating a Neon project. 

Technically, this is a part of Console release. Should we mention it twice?

Reply:
Good point. No need to mention it twice. Let's leave that line as this:
"Added support for PostgreSQL 15.0 and PostgreSQL 15 extensions supported by Neon."